gallery/src/components/NxTable/NxTableFilterExample.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTable,
+  NxTableBody,
+  NxTableCell,
+  NxTableHead,
+  NxTableRow,
+  NxFilterInput
+} from '@sonatype/react-shared-components';
+import { toLower } from 'ramda';
+
+const initialState = [
+  {name: 'Anna', country: 'USA'},
+  {name: 'Lean', country: 'France'},
+  {name: 'Louis', country: 'France'},
+  {name: 'Zach', country: 'Colombia'},
+  {name: 'Jimmy', country: 'Germany'}
+];
+
+const NxTableFilterExample = () => {
+
+  const [rows, setRows] = useState(initialState);
+  const [nameFilter, setNameFilter] = useState('');
+  const [countryFilter, setCountryFilter] = useState('');
+
+  const dropdownDefaultLabel = 'All countries';
+  const listId = 'countryList';
+  const countries = new Set(initialState.map((state: any) => {
+    return state['country'];
```

Comment:
Better variable name:
```suggestion
  const countries = new Set(initialState.map((row: any) => {
    return row['country'];
```